alternator/executor.cc

```diff
@@ -3118,7 +3118,7 @@ static void check_big_array(const rjson::value& val, int& size_left) {
     }
 }
 
-static void check_big_object(const rjson::value& val, int& size_left) {
+static void check_big_object(const rjson::value& val, size_t& size_left) {
     size_left -= 10 * val.MemberCount();
     for (const auto& m : val.GetObject()) {
         if (size_left < 0) {
```

Comment:
This check now will never be true after `size_left` was changed to `size_t`.

Reply:
Good catch. I wonder that the tests didn't catch it.